### PR TITLE
Image Upload Adjustments

### PR DESCRIPTION
**Pull request by MatkoTom**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant
motivation and context. List any dependencies that are required for this change.

Updated logic for settings image loading.

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can
reproduce. Please also list any relevant details for your test configuration. All testing was done
manually by following, plainly, functionality requirements. If there are some special edge cases or
special ways that things need to be tested they will be mentioned below.

Dev testing

**Test Configuration**:

* Firmware version: A556BXXS3AXF2
* Hardware: SAMSUNG Galaxy A55
* SDK: Android 14

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: app/src/main/res/layout/profile_picture.xml [hunk 1]
```diff
@@ -25,7 +25,7 @@
         android:background="@color/dark_background_transp"
         android:backgroundTint="@color/background_80_transp"
         android:elevation="@dimen/five_dp_margin"
-        android:visibility="gone"
+        android:visibility="visible"
```
Comment (ivolarevic) [inline app/src/main/res/layout/profile_picture.xml:28]: Why did you change visibility? Did you test this?

**Review by ivolarevic — APPROVED** [review]
LGTM

File: app/src/main/res/layout/profile_picture.xml (hunk 1 above)
Comment (MatkoTom) [reply app/src/main/res/layout/profile_picture.xml:28]: Tested and discussed.
